### PR TITLE
yc-1070 checkbox login

### PR DESCRIPTION
**Pull request by AlexandreJunod**

### CODE REVIEW

File: geocity/apps/accounts/forms.py [hunk 1]
```diff
@@ -166,6 +167,14 @@ def __init__(self, *args, **kwargs):
         ),
     )
     captcha = CaptchaField(required=True)
+    policy = forms.BooleanField(
+        required=True,
+        label=mark_safe(
+            _(
+                'J\'ai lu et j\'accepte la <b><a href="https://geocity.ch/legal/confidentialite.pdf" target="_blank">politique de confidentialité</a></b>'
```
Comment (rbovard) [inline geocity/apps/accounts/forms.py:174]: You need to use the Constance value (like in the footer), not hardcode it

Comment (monodo) [reply geocity/apps/accounts/forms.py:174]: +1

**Review by monodo — DISMISSED** [review]

File: geocity/apps/accounts/forms.py [hunk 2]
```diff
@@ -171,7 +172,7 @@ def __init__(self, *args, **kwargs):
         required=True,
         label=mark_safe(
             _(
-                'J\'ai lu et j\'accepte la <b><a href="https://geocity.ch/legal/confidentialite.pdf" target="_blank">politique de confidentialité</a></b>'
+                f'J\'ai lu et j\'accepte la <b><a href="{config.PRIVACY_POLICY_URL}" target="_blank">politique de confidentialité</a></b>'
```
Comment (rbovard) [inline geocity/apps/accounts/forms.py:175]: What happens if not defined?

https://github.com/yverdon/geocity/blob/develop/geocity/apps/core/templates/base_generic.html#L220-L223

And are you sure you want to put the link in bold?

Comment (AlexandreJunod) [reply geocity/apps/accounts/forms.py:175]: It opens the same page again. In bold it's easier to understand this is a link, and shows how important it is

**Pull request merged**